e2e/wdio/platform/pages/input.po.ts

```diff
@@ -0,0 +1,33 @@
+import { BaseComponentPo } from './base-component.po';
+import { webDriver } from '../../driver/wdio';
+
+export class InputPo extends BaseComponentPo {
+    readonly url = '/input';
+    readonly root = '#page-content';
+
+    defaultInput = '#input1';
+    textInput = '#input2';
+    numberInput = '#input3';
+    compactInput = '#input4';
+    readonlyInput = '#input5';
+    disabledInput = '#input6';
```

Comment:
is there any way not to use the random-gen IDs? or are you sure these will be stable?